src/core/sync/event.d

```diff
@@ -0,0 +1,318 @@
+/**
+ * The event module provides a primitive for lightweight signalling of other threads
+ * (aka Windows Events)
+ *
+ * Copyright: Copyright (c) 2019 D Language Foundation
+ * License: Distributed under the
+ *    $(LINK2 http://www.boost.org/LICENSE_1_0.txt, Boost Software License 1.0).
+ *    (See accompanying file LICENSE)
+ * Authors: Rainer Schuetze
+ * Source:    $(DRUNTIMESRC core/sync/event.d)
+ */
+module core.sync.event;
+
+version (Windows)
+{
+    import core.sys.windows.basetsd /+: HANDLE +/;
```

Comment:
Why these comments?

Reply:
This follows the example of https://github.com/dlang/druntime/pull/2400/files (selective imports create a public alias).

Reply:
Hasn't this been fixed a year ago or so?

Reply:
Oh well, looks like it's really still in deprecation, but there's a PR to remove them for good:

https://github.com/dlang/dmd/pull/9393

Reply:
That will go in very soon.

Reply:
@rainers this has been merged now.

Reply:
Not good  enough:
```
test.d(3): Deprecation: core.sync.event.HANDLE is not visible from module test
```

Reply:
😞